objects/src/accounts/seed.rs

```diff
@@ -47,7 +47,9 @@ pub fn get_account_seed(
 
     #[allow(unused_variables)]
     let (digest, seed) = recv.recv().unwrap();
-    let _ = stop.set(true);
+
+    // Safety: this is the only writer for this lock, it should never be poisoned
+    *stop.write().unwrap() = true;
```

Comment:
The guard is dropped after the the assignment happens, because this is not a named variable [ref](https://doc.rust-lang.org/reference/destructors.html?highlight=drop#temporary-scopes)